src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+
+        ball.setColor(ColorSupplier.getRandomColor());
+        ball.setNumber(random.nextInt(100));
+
+        return ball;
```

Comment:
Avoid using static methods as per the checklist. Instead, you can create an instance of the `Lottery` class in your application and use its method to get a random ball.